body-segmentation/package.json

```diff
@@ -46,7 +47,7 @@
     "ts-node": "~8.8.2",
     "tslint": "~6.1.3",
     "tslint-no-circular-imports": "~0.7.0",
-    "typescript": "~3.9.9",
+    "typescript": "~4.4.0",
```

Comment:
Nit: We should update to 5.0.4 if possible, but this isn't strictly necessary.

Reply:
Actually, that can wait for tfjs 4.10.0 (or can be done now. it shouldn't really affect the demo).